src/tally.h

```diff
@@ -43,13 +48,35 @@ namespace pyne
     /// \param entity_size the physical size of the tally volume
     /// \param normalization the number required to normalize your tally
     Tally(std::string type, std::string particle_name, int entity,
-    std::string entity_type, std::string entity_name,
-    std::string tally_name = "", double entity_size = 0.0,
-    double normalization = 1.0);
-
-    ~Tally (); /// default destructor
-
-
+          std::string entity_type, std::string entity_name,
+          std::string tally_name = "", double entity_size = 0.0, 
+          double normalization = 1.0);
+ 
+    /// Constructor from passed in vars
```

Comment:
Better comment to indicate how these two constructors are different: one is for a cell/surface-based tally and the other is for a mesh tally.